mimic/rest/maas_api.py

```diff
@@ -935,6 +973,17 @@ def list_alarm_examples(self, request, tenant_id):
         request.setResponseCode(200)
         return json.dumps({'values': axs, 'metadata': metadata})
 
+    @app.route('/v1.0/<string:tenant_id>/alarm_examples/<string:metric>', methods=['POST'])
+    def alarm_examples_template(self, request, tenant_id, metric):
+        """
+        specific alarm example template
```

Comment:
Specific to what? What kind of "template"?  Is there actually an endpoint called `alarm_examples` in MAAS?